libs/api/domains/syslumenn/src/lib/client/syslumenn.client.ts

```diff
@@ -20,30 +26,57 @@ export class SyslumennClient {
     private clientConfig: SyslumennClientConfig,
   ) {}
 
-  private async getToken() {
+  private async login() {
     const config = {
       notandi: this.clientConfig.username,
       lykilord: this.clientConfig.password,
     }
 
     const response: { data: ILogin } = await this.httpService
-      .post(`${this.clientConfig.url}/dev/v1/Innskraning`, config)
+      .post(`${this.clientConfig.url}/v1/Innskraning`, config)
```

Comment:
I removed `/dev` from this url, since we are reusing the login funcitonality. 
This means we will have to update the value in helm. Both I think we would have to do that in the end anyways.

Reply:
Nice. Can you make the changes in the helm repo? I guess you'll need to change the host in `portal` and add all the variables in `application-system`

Reply:
Of course 😄  I was thinking of creating a single pr in helm where I update it after this one get's merged. How does that sound? :)

Reply:
Great!